tests/Endpoints/DocumentsTest.php

```
@@ -358,31 +355,26 @@ public function testAddDocumentsCsvInBatchesWithDelimiter(): void
-                          self::assertSame(["id;title\n235115704;Mister Klein", null, ';'], [$documents, $primaryKey, $delimiter]);
-
-                          return MockTask::create(TaskType::DocumentEdition);
-                      default:
-                          self::fail();
-                  }
-              });
-
+        $http = $this->createMock(Http::class);
+        $http->expects(self::exactly(2))
+            ->method('post')
+            ->willReturnCallback(function (string $path, $documents, $query, $contentType): array {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$path'. (undefined)

(UnusedFormalParameter)